Add a script to list woo/DEWPed versions (#1774)
woocommerce/google-listings-and-ads: Working with DEWP.md

To help with that we implemented the `extracted/` prefix. It's also a custom implementation in [/webpack.config.js](/develop/webpack.config.js).
Thanks to that even though a package is bundled, the given import would fetch it from external.

## NPM scripts

We also have a bunch of NPM scripts to help us work with them daily.

### Outdated

In a regular project to track outdated packages, you would use `npm outdated`. However, with DEWP updating a bundled package is not the same as updating externalized one. For example, if we aim to support WC L-2, we theoretically need to test our extensions with all the package versions used by that version, and we cannot update it to the latest as we go. So we have two scripts to separate those cases:

- `npm run outdated:dewp` Check for outdated packages that are listed in `.externalized.json`
- `npm run outdated:nondewp` Check for those not listed there.

### What is the version of the DEWPed package?

As we externalize packages we lose control of the version of the package that will run in the wild (on WordPress instance).

> However, with the current platform, we not only loos control, but we also lose the information of what is actually the range of package we should expect.

Reviewer: ```suggestion
However, with the current platform, we not only lose control, but we also lose the actual information on the range of package we should expect.
```

So, even if we know we support WooCommerce 6.9+, what are the `@woocommerce/components` there?

There is no published table of those, and finding them usually requires quite a lot of manual digging. To mitigate that we created another script.

- `npm run dewps:woo 6.9.4` - where `6.9.4` is the version of WooCommerce you would like to check.

> Please note this simple script still have several limitations.

Reviewer: ```suggestion
Please note this simple script still has several limitations.
```
